students/john_navitsky/session04/trigrams.py

```diff
@@ -17,20 +50,52 @@
 		pairs[pair]=[]
 	pairs[pair].append(follower)
 
-print("PAIRS:",pairs)
+#print()
+#print("PAIRS:",pairs)
 
 i=1
 out_list=list(random.choice(list(pairs.keys())))
-print("SEED:",out_list)
+#print()
+#print("SEED_PAIR:",out_list)
 
+#print()
+#print("writing the story",end="")
 while True:
```

Comment:
more tabs -- you REALLY need to fix your editor!

any editor wirth its salt will be able to put in spaces when you hit the <tab> key.

in Sublime it's:   `"translate_tabs_to_spaces": true`

and I think the Anaconda package for Sublime turns it on by default.